HaxeManual/06-language-features.tex

```diff
@@ -455,6 +455,22 @@ \subsection{In the Haxe Standard Library}
 \trivia{``using'' using}{Since the \expr{using} keyword was added to the language, it has been common to talk about certain problems with ``using using'' or the effect of ``using using''. This makes for awkward English in many cases, so the author of this manual decided to call the feature by what it actually is: Static extension.}
 
 
+\subsection{Static Extension Metadata}
+\label{lf-static-extension-metadata}
+\since{4.0.0}
+
+\paragraph{Exclusion from Static Extension}
+
+By default, using a class as a static extension brings all of its static methods into the context, allowing them to be used as extensions of the appropriate types. In certain situations, the class can provide other static methods which are not intended for static extension. To make sure they do not interfere with the proper methods of the type, these methods can be marked with \expr{@:noUsing}:
+
+\haxe{assets/StaticExtension3.hx}
+
+\paragraph{Default Static Extension}
+
+It is also possible to always enable particular static extensions for a given type, by annotating the type with the \expr{@:using(args...)} metadata. The arguments are the full dot paths of static extension classes that will be applied on the type:
```

Comment:
This implies that `@:using` works for every type, while it seems to basically only work for enums and classes at the moment...

https://github.com/HaxeFoundation/haxe/issues/8022
https://github.com/HaxeFoundation/haxe/issues/7930
https://github.com/HaxeFoundation/haxe/issues/7859

Reply:
I guess we'll leave this as-is for now :)